app/config/permissions.neon

```diff
@@ -12,6 +12,9 @@ permissions:
     - allow: true
       role: superadmin
 
+    - allow: true
+      role: scope-master
```

Comment:
Maybe I misunderstood something, but wouldn't this allow all permission to all users by default? If you are setting scope-master to all new login issued tokens?

Reply:
Sorry my bad, wrongly interpreted this line https://github.com/ReCodEx/api/pull/177/commits/ed1493bdd00bd7d6104f00f98df1d732255742e1#diff-acd2f5cdd382ea94a69ecdddce42c7f9R42